### PR TITLE
Implement SPV_KHR_uniform_group_instructions extension

### PR DESCRIPTION
**Pull request by KornevNikita**
Spec: https://github.com/KhronosGroup/SPIRV-Registry/pull/138

### CODE REVIEW

File: include/spirv/unified1/spirv.core.grammar.json
Comment (bashbaug) [inline include/spirv/unified1/spirv.core.grammar.json:8589]: Can you please only include the "extensions" tag on the capability, and not on the instructions?  Rationale being that if the capability is gated by the extension, any instruction that depends on the capability will also be implicitly gated by the extension.

This change will apply to this instruction and all of the instructions below added in this PR.

**Review by bashbaug — CHANGES_REQUESTED** [review]
Discussed in the March 2nd teleconference.  This change mostly looks good, but can you please make one small change to the extension tagging for the instructions added in this PR?

File: include/spirv/unified1/spirv.core.grammar.json
Comment (KornevNikita) [reply include/spirv/unified1/spirv.core.grammar.json]: Thanks, done https://github.com/KhronosGroup/SPIRV-Headers/pull/269/commits/7744288e2d4a9c949ed0b6588a603c963551e025

**Review by bashbaug — COMMENTED** [review]
Thanks, latest changes LGTM.

Tagging @raunraun and @johnkslang - I think we agreed to merge these changes after this fix in yesterday's teleconference, but I'd prefer to have a non-Intel approval before merging.

**Comment by raunraun** [comment]
LGTM! - rerunning the checkers now

**Comment by KornevNikita** [comment]
@raunraun should we wait for https://github.com/KhronosGroup/SPIRV-Registry/pull/138 merge or this PR can be merged now?

**Pull request merged**
